src/components/atoms/tooltip/tooltip.md

```diff
@@ -0,0 +1,19 @@
+```js props
+<Tooltip {props} />
```

Comment:
Can you also add some text to the tooltip, the example looks incomplete without it 😄 

This should work:
```jsx
<Tooltip {props}>Here is some text</Tooltip>
```